Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl

```diff
@@ -157,8 +157,8 @@ reagent-desc-hooch = Either someone's failure at cocktail making or attempt in a
 reagent-name-iced-beer = iced beer
 reagent-desc-iced-beer = A beer which is so cold the air around it freezes.
 
-reagent-name-irish-car-bomb = irish car bomb
-reagent-desc-irish-car-bomb = A troubling mixture of irish cream and ale.
+reagent-name-irish-slammer = irish slammer
+reagent-desc-irish-slammer = An unconventional mixture of irish cream and stout.
```

Comment:
Reading through the YAML, this drink is actually wrong anyway.

A Car Bomb/Irish Slammer/Dublin Drop/etc is made with _three_ ingredients - you layer a shot glass with irish cream and Irish whiskey like Jameson's, then drop it into 3/4s of a pint of something like Guinness.

The game appears to model this  by "whiskey + cream = irish cream", "irish cream + ale" = the cocktail.

To be a bit of a snob, I'm not sure if there's any advantage at all to having this drink in-game if it's not actually modelling the cocktail! This is a party drink all about being forced to chug the entire drink in a few seconds to avoid the base stout curdling the cream.